docs/03-node-operators/configurations/fiatStableMinGasPrice/fiat-stable-min-gas-price.md

```diff
@@ -0,0 +1,146 @@
+
+### **🔗 HTTP Call (Off-Chain Method)**
+
+This approach uses a **centralized or semi-centralized API** to fetch the current **RBTC-to-fiat price** and then calculate a gas price based on that.
+
+### How it works:
+
+- The system (like a wallet, dApp backend, or node) sends an **HTTP request** to a price oracle or API like:
+    - CoinGecko
+    - Chainlink's external adapters
+    - Custom backend service
+- It retrieves the current **RBTC/USD price**, then calculates:
+  `minGasPrice = fiatMinPrice / ethPrice`
+- Example: If you want the gas price to represent $0.01, and RBTC = $2000, then:
+  `minGasPrice = 0.01 / 2000 = 0.000005 RBTC (or 5 gwei)`
+
+### ✅ Pros:
+
+- Simple, flexible, easy to integrate.
+- Can pull from a wide range of data sources.
+
+### ❌ Cons:
+
+- **Centralization risk** – you're trusting the HTTP source.
+- **Not trustless** – the blockchain doesn't verify the result.
+- **Latency and reliability** – dependent on network and endpoint availability.
+
+---
+
+### ⛓️ **On-Chain Oracle Call (Decentralized/Trustless Method)**
+
+This approach fetches the **RBTC-to-fiat price from a smart contract**, often powered by a **decentralized oracle network** like **Chainlink**.
+
+### How it works:
+
+- Smart contracts call an **on-chain oracle** (e.g., Chainlink RBTC/USD price feed).
+- The gas price is derived using the oracle's latest reported price.
+- This is done **inside smart contracts**, ensuring consistency and trust.
+
+### ✅ Pros:
+
+- **Trustless and secure** – all nodes agree on the same value.
+- **Tamper-resistant** – oracle data is resistant to manipulation.
+- **Fully on-chain** – compatible with DeFi, DAO governance, etc.
+
+### ❌ Cons:
+
+- **Gas cost** – reading from an oracle contract uses gas.
+- **Oracle update frequency** – price may not always be up to the second.
+- **Complexity** – needs integration with oracle networks and possibly fallback logic.
+
+### Summary Table:
+
+| Method | Trust Level | Cost | Speed | Use Case |
+| --- | --- | --- | --- | --- |
+| HTTP API (Off-chain) | Centralized | Low | Fast | Backend services, UI, analytics |
+| On-chain Oracle | Decentralized | Moderate | Near real-time | Smart contracts, DeFi, DAOs |
+
+### RSK Configuration
+
+[https://github.com/rsksmart/rskj/pull/2310/files#diff-242a8c6b5d9003504710db696af1f94412a52cdbd3369df55cff4be2ea8697eaR204-R234](https://github.com/rsksmart/rskj/pull/2310/files#diff-242a8c6b5d9003504710db696af1f94412a52cdbd3369df55cff4be2ea8697eaR204-R234)
+
+You can see in master that this is a miner configuration and looks like this:
+
+**HTTP CALL**
+
+Will use HTTP to request the min gas price
+
+```yaml
+miner {
+    stableGasPrice {
+        enabled = false
+
+        source {
+	        # method options: HTTP_GET | ETH_CALL
+					# Examples:
+					method = "HTTP_GET"
+					params {
+						url = "https://domain.info/ticker"
+						jsonPath = "/USD/buy"
```

Comment:
I think it would be helpful to include a brief explanation of the different parameters. For example, jsonPath might not be obvious to all users

Reply:
Addressed